scripts/run_console_local.sh

```diff
@@ -14,38 +23,49 @@ add_host_port_arg (){
 }
 
 pull_ocp_console_image (){
-   docker pull quay.io/openshift/origin-console:latest
+   $POD_MANAGER pull quay.io/openshift/origin-console:latest
 }
 
-run_docker_console (){
+run_ocp_console_image (){
     secretname=$(kubectl get serviceaccount default --namespace=kube-system -o jsonpath='{.secrets[0].name}')
     endpoint=$(kubectl config view -o json | jq '{myctx: .["current-context"], ctxs: .contexts[], clusters: .clusters[]}' | jq 'select(.myctx == .ctxs.name)' | jq 'select(.ctxs.context.cluster ==  .clusters.name)' | jq '.clusters.cluster.server' -r)
 
     echo -e "Using $endpoint"
-    command -v docker run -it $args \
+    $POD_MANAGER run -dit $args \
       -e BRIDGE_USER_AUTH="disabled" \
       -e BRIDGE_K8S_MODE="off-cluster" \
       -e BRIDGE_K8S_MODE_OFF_CLUSTER_ENDPOINT=$endpoint \
       -e BRIDGE_K8S_MODE_OFF_CLUSTER_SKIP_VERIFY_TLS=true \
       -e BRIDGE_K8S_AUTH="bearer-token" \
       -e BRIDGE_K8S_AUTH_BEARER_TOKEN=$(kubectl get secret "$secretname" --namespace=kube-system -o template --template='{{.data.token}}' | base64 --decode) \
       quay.io/openshift/origin-console:latest &> /dev/null
+}
 
-    docker_exists=${?}; if [[ ${docker_exists} -ne 0 ]]; then
-        echo -e "${BLUE}The OLM is accessible via web console at:${RESET}"
-        echo -e "${BLUE}https://localhost:9000/${RESET}"
+verify_ocp_console_image (){
+  while true; do
```

Comment:
What's the purpose of the loop? when I tried this on mac it succeeded at first, then slept 10s and hit the else block and exited.

Reply:
maybe I'm missing something but it shouldn't be hitting the else - also tested on mac and linux:
https://asciinema.org/a/WLQ5sQToBod41aQTeTEWPnh13

loops message so that session is blocked with cleanup on quit vs. script runs, exits, and user forgets to cleanup manually.